pom.xml

```diff
@@ -767,11 +766,11 @@
                     </executions>
                     <configuration>
                         <tasks>
-                            <echo>Copying additional assets for DMG</echo>
-                            <copy file="${basedir}/src/main/resources/Localizable.strings" tofile="${basedir}/target/${project.build.finalName}/${project.name}.app/Contents/Resources/en.lproj/Localizable.strings"/>
-                            <copy file="${basedir}/src/main/nsis/config.properties" tofile="${basedir}/target/${project.build.finalName}/${project.name}.app/Contents/Resources/conf/config.properties"/>
-                            <replace file="${basedir}/target/${project.build.finalName}/${project.name}.app/Contents/Resources/conf/config.properties" token="#placeholder" value="${config.additional.properties}"/>
-                            <replace file="${basedir}/target/${project.build.finalName}/${project.name}.app/Contents/Resources/conf/config.properties" token="\n" value="${line.separator}"/>
+                          <echo>Copying additional assets for DMG</echo>
+                          <copy file="${basedir}/src/main/resources/Localizable.strings" tofile="${basedir}/target/${project.build.finalName}/${project.name}.app/Contents/Resources/en.lproj/Localizable.strings"/>
+                          <copy file="${basedir}/src/main/nsis/config.properties" tofile="${basedir}/target/${project.build.finalName}/${project.name}.app/Contents/Resources/conf/defaultConfig.properties"/>
+                          <replace file="${basedir}/target/${project.build.finalName}/${project.name}.app/Contents/Resources/conf/defaultConfig.properties" token="#placeholder" value="${config.additional.properties}"/>
+                          <replace file="${basedir}/target/${project.build.finalName}/${project.name}.app/Contents/Resources/conf/defaultConfig.properties" token="\n" value="${line.separator}"/>
```

Comment:
This `defaultConfig.properties` is used as a default and will be copied to user specific config dir during runtime if they don't have `config.properties` yet.